jdaviz/configs/specviz/tests/test_helper.py

```diff
@@ -208,38 +208,29 @@ def test_get_spectral_regions_unit_conversion(specviz_helper, spectrum1d):
 
     # Mouseover without data should not crash.
-    spec_viewer.on_mouse_or_key_event({'event': 'mousemove', 'domain': {'x': 6100, 'y': 12.5}})
-    assert spec_viewer.label_mouseover.pixel == ''
-    assert spec_viewer.label_mouseover.world_label_prefix == '\xa0'
-    assert spec_viewer.label_mouseover.world_ra == ''
-    assert spec_viewer.label_mouseover.world_dec == ''
-    assert spec_viewer.label_mouseover.world_label_prefix_2 == '\xa0'
-    assert spec_viewer.label_mouseover.world_ra_deg == ''
-    assert spec_viewer.label_mouseover.world_dec_deg == ''
-    assert spec_viewer.label_mouseover.icon == ''
+    label_mouseover = specviz_helper.app.session.application._tools['g-coords-info']
+    label_mouseover._viewer_mouse_event(spec_viewer,
+                                        {'event': 'mousemove', 'domain': {'x': 6100, 'y': 12.5}})
+    assert label_mouseover.as_text() == ('', '', '')
+    assert label_mouseover.icon == ''
 
     # If the reference (visible) data changes via unit conversion,
     # check that the region's units convert too
     specviz_helper.load_spectrum(spectrum1d)
 
     # Also check coordinates info panel.
     # x=0 -> 6000 A, x=1 -> 6222.222 A
-    spec_viewer.on_mouse_or_key_event({'event': 'mousemove', 'domain': {'x': 6100, 'y': 12.5}})
-    assert spec_viewer.label_mouseover.pixel == '6.10000e+03, 1.25000e+01'
-    assert spec_viewer.label_mouseover.world_label_prefix == 'Wave'
-    assert spec_viewer.label_mouseover.world_ra == '6.00000e+03 Angstrom (0 pix)'  # actual: 0.4
-    assert spec_viewer.label_mouseover.world_label_prefix_2 == 'Flux'
-    assert spec_viewer.label_mouseover.world_ra_deg == '1.24967e+01 Jy'
-    assert spec_viewer.label_mouseover.icon == 'a'
-    spec_viewer.on_mouse_or_key_event({'event': 'mousemove', 'domain': {'x': None, 'y': 12.5}})
-    assert spec_viewer.label_mouseover.pixel == ''
-    assert spec_viewer.label_mouseover.world_label_prefix == '\xa0'
-    assert spec_viewer.label_mouseover.world_ra == ''
-    assert spec_viewer.label_mouseover.world_dec == ''
-    assert spec_viewer.label_mouseover.world_label_prefix_2 == '\xa0'
-    assert spec_viewer.label_mouseover.world_ra_deg == ''
-    assert spec_viewer.label_mouseover.world_dec_deg == ''
-    assert spec_viewer.label_mouseover.icon == 'a'
+    label_mouseover._viewer_mouse_event(spec_viewer,
+                                        {'event': 'mousemove', 'domain': {'x': 6100, 'y': 12.5}})
+    assert label_mouseover.as_text() == ('Cursor 6.10000e+03, 1.25000e+01',
+                                         'Wave 6.00000e+03 Angstrom (0 pix)',  # actual 0.4
+                                         'Flux 1.24967e+01 Jy')
+    assert label_mouseover.icon == 'a'
+
+    label_mouseover._viewer_mouse_event(spec_viewer,
+                                        {'event': 'mousemove', 'domain': {'x': None, 'y': 12.5}})
+    assert label_mouseover.as_text() == ('', '', '')
+    assert label_mouseover.icon == ''
```

Comment:
Should still be A according to the old test. Is this new behavior intended?
```suggestion
    assert label_mouseover.icon == 'a'
```

Reply:
This is intended, the icon clears when leaving the valid range.  I think previously the traitlet remained set but the UI logic skipped showing it.